.github/contributing/commits.md

```diff
@@ -1,9 +1,8 @@
 # Commits
 
-Commits must follow the
-[Conventional Commits](https://www.conventionalcommits.org/en/v1.0.0/#summary) syntax.
-The `CHANGELOG.md` file is generated by an automated process using standard version that requires the commits
-to be in a specific format, so they can be parsed.
+All commits must follow the
+[Conventional Commits](https://www.conventionalcommits.org/en/v1.0.0/#summary) specification syntax.
+The `CHANGELOG.md` file is generated automatically using standard version that requires the commits to be in a specific format in order to be parsed.
```

Comment:
Might be a good idea to add the link to `standard-version` plugin.

```suggestion
The `CHANGELOG.md` file is generated automatically using [standard-verison](https://www.npmjs.com/package/standard-version), which requires the commits to be in a specific format in order to be parsed.
```

- [ ] Add link to `standard-verison`